app/views/poms/index.html.erb

```diff
@@ -1,9 +1,41 @@
 <% content_for :switcher do %>
   <%= render '/layouts/prison_switcher' %>
 <% end %>
+<h1 class="govuk-heading-l">Manage your staff</h1>
 
-<% if @prison.womens? %>
-  <%= render 'female_index' %>
-<% else %>
-  <%= render 'male_index' %>
-<% end %>
+<div class="govuk-tabs" data-module="govuk-tabs">
+  <h2 class="govuk-tabs__title">Manage your staff</h2>
+
+  <ul class="govuk-tabs__list" role='tablist'>
+    <li class="govuk-tabs__list-item govuk-tabs__list-item--selected">
+      <a class="govuk-tabs__tab" href="#active_probation_poms">
+        Active Probation officer POMs (<%= @poms.count { |pom| %w[active unavailable].include?(pom.status) && pom.probation_officer? } %>)
```

Comment:
### Opportunity to DRY up

It seems a shame to be duplicating this special filtering logic throughout this view. We perform the same filter twice: once for the count, and once to render the list.

Previously splitting was performed in the Controller, so we had:
`@active_probation_poms`, `@active_prison_poms` and `@inactive_poms`
These were nice & easy to use in the view – but I imagine your intention is for the View layer to perform this filtering (since it's only done for display purposes)?

Could this filtering logic be put into helper methods instead of cluttering the view?

`active_probation_poms`, `active_prison_poms` and `inactive_poms` could conceivably become helper methods...